app.js

```diff
@@ -297,6 +415,18 @@ app.post('/hubs/:hubSlug/start_activity', function(req, res){
   }
 })
 
+app.post('/hubs/:hubSlug/devices/:deviceSlug/activate_command', function(req, res){
```

Comment:
Yup, I'm definitely in favor of that. I also agree with @novagl that the overall activity status is a bit bloated.